lib/soapy_cake/admin_batched.rb

```diff
@@ -43,7 +40,18 @@ def to_enum
 
       private
 
-      def next_batch
+      def fetch_elements(enumerator)
```

Comment:
`fetch_elements`? That doesn't sound any different than `fetch_batch`... Wouldn't `yield_elements` be better?